podman_compose.py

```diff
@@ -376,7 +380,7 @@ def assert_volume(compose, mount_dict):
     inspect volume to get directory
     create volume if needed
     """
-    if mount_dict["type"] != "volume": return
+    if mount_dict["type"] != "volume" or mount_dict["external"]: return
```

Comment:
Here I get a `KeyError: 'external'` for some volumes.
I changed it to `mount_dict.get("external")` and it seems to work, but maybe there is a better way to produce a boolean.